### ISSUE TITLE
Naming Convention for Server Name and Generated Files

### ISSUE THREAD

**Issue by KarthikAbiram** [ni/grpc-labview#36]
A low priority request/question:

What would be the recommended naming convention for the server name and generated folders & files? If there are multiple naming conventions that are being followed for various types, it would be helpful to document them as recommended guidelines.

Looking at the folder names in parallel, I had named my server to be in "Normal Case with Spaces". In the recent code base, there is an auto-suffix for server and client which are of the formats - "_server" and "_client", which ends up with mixed naming convention for the generated library.

![image](https://user-images.githubusercontent.com/50194596/141732099-85a6e283-a8dd-487d-b098-aedecf614d9e.png)
![image](https://user-images.githubusercontent.com/50194596/141732174-f0a77afe-9bea-4349-bc79-d00386e06006.png)

Similarly in gRPC lv support, 
![image](https://user-images.githubusercontent.com/50194596/141732818-14a3aa27-d417-4e4a-8501-a6c3fd775ba4.png)
![image](https://user-images.githubusercontent.com/50194596/141732831-1de436c2-6b61-4cc2-9587-2a44c70de077.png)

If we can choose a specific naming convention throughout, it would be helpful for consistency :)
